app/src/lib/stores/app-store.ts

```diff
@@ -1979,7 +1981,7 @@ export class AppStore {
         const hasValidToken =
           account.token.length > 0 ? 'has token' : 'empty token'
         log.info(
-          `[getAccountForRemoteURL] account found for remote: ${remote} - ${account.login} (${hasValidToken})`
+          `[AppStore.getAccountForRemoteURL] account found for remote: ${remote} - ${account.login} (${hasValidToken})`
```

Comment:
A little bit of readability in here because we don't get the context for these when logging to disk